src/views/FavoriteFoodMain.js

```diff
@@ -1,128 +1,48 @@
 import React from 'react';
-import FavoriteFoods from './FavoriteFoods';
+import SelectedFood from './SelectedFood';
+import FoodMain from './FoodMain';
 import jsonData from '../assets/data/data.json';
-import CardColumns from 'react-bootstrap/CardColumns';
 
 
 class FavoriteFoodMain extends React.Component {
 
   constructor(props){
     super(props);
     this.state = {
-      gallery: jsonData
+      allFavoriteFoods: jsonData,
+      displayModal: false,
+      selectedFavoriteFood: {}
+     
     }
   }
   
+displayAsModel = (foodName) => {
```

Comment:
watch out for this typo in Modal

Reply:
Right on. I did not check that and I should have haah. @selmaT273